tutorials/Qubit/Qubit.ipynb

```diff
@@ -0,0 +1,309 @@
+    "\n",
+    "$$\\begin{bmatrix} \\alpha \\\\ \\beta \\end{bmatrix} =\n",
+    "\\begin{bmatrix} \\alpha \\\\ 0 \\end{bmatrix} + \\begin{bmatrix} 0 \\\\ \\beta \\end{bmatrix} =\n",
+    "\\alpha \\cdot \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} + \\beta \\cdot \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix}$$\n",
+    "\n",
+    "Because of this, these two states are known as **basis states**.\n",
+    "\n",
+    "These two vectors have two additional properties. First off, as mentioned before, both are **normalized**:\n",
+    "\n",
+    "$$\\langle \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} , \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} \\rangle =\n",
+    "\\langle \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix} , \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix} \\rangle = 1$$\n",
+    "\n",
+    "Second, they are **orthogonal** to each other:\n",
+    "\n",
+    "$$\\langle \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} , \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix} \\rangle =\n",
+    "\\langle \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix} , \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} \\rangle = 0$$\n",
+    "\n",
+    "> As a reminder, $\\langle V , W \\rangle$ is the [inner product](../LinearAlgebra/LinearAlgebra.ipynb#Inner-Product) of $V$ and $W$.\n",
+    "\n",
+    "This means that these vectors form an **orthonormal basis**. To be more specific, this is the **computational basis**, also known as the **canonical basis**.\n",
+    "\n",
+    "> There exist other orthonormal bases, such as, for example, the **Hadamard basis**:\n",
+    ">\n",
+    "> $$\\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} \\end{bmatrix} \\text{and} \\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ -\\frac{1}{\\sqrt{2}} \\end{bmatrix}$$\n",
+    ">\n",
+    "> $$\\begin{bmatrix} \\alpha \\\\ \\beta \\end{bmatrix} =\n",
+    "\\frac{\\alpha + \\beta}{\\sqrt{2}} \\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} \\end{bmatrix} +\n",
+    "\\frac{\\alpha - \\beta}{\\sqrt{2}} \\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ -\\frac{1}{\\sqrt{2}} \\end{bmatrix}$$\n",
+    ">\n",
+    "> The Hadamard basis is widely used in quantum computing, for example in the [BB84 quantum key distribution protocol](https://en.wikipedia.org/wiki/BB84)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dirac Notation\n",
+    "\n",
+    "Completely writing out each vector when doing quantum calculations already takes up a lot of space, and this will get even worse when we introduce quantum gates and multi-qubit systems. Luckily, there is a way around this using **Dirac notation**. In Dirac notation, a vector is represented by a symbol called a **ket**. For example, a qubit in state $0$ is represented by the ket $|0\\rangle$, and a qubit in state $1$ is represented by the ket $|1\\rangle$:\n",
```

Comment:
```suggestion
    "Writing out each vector when doing quantum calculations takes up a lot of space, and this will get even worse once we introduce quantum gates and multi-qubit systems. **Dirac notation** is a shorthand notation that helps solve this issue. In Dirac notation, a vector is denoted by a symbol called a **ket**. For example, a qubit in state $0$ is represented by the ket $|0\\rangle$, and a qubit in state $1$ is represented by the ket $|1\\rangle$:\n",
```